xml/book_poc_crowbar.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<book xmlns="http://docbook.org/ns/docbook"
+      xmlns:xi="http://www.w3.org/2001/XInclude"
+      xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:lang="en"
+      xml:id="book.poc.crowbar">
+ <title>&crow; Proof of Concept Guide</title>
+ <info>
+ <xi:include href="copyright_suse_cloud.xml"/>
+  <abstract>
+   <para>
+     At the time of the &productnumber; release, this guide
+     is a proof of concept (POC) of a &productname; deployment.
```

Comment:
is a proof of concept (POC) of a &productname; deployment.

should be an &productname;